data.sql

```diff
@@ -1,5 +1,19 @@
-insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
-values ('Agumon', '2020-02-03', 0, '1', 10.23),
-('Gabumon', '2018-11-15', 2, '1', 8),
-('Pikachu', '2021-01-7', 1, '0', 15.04),
-('Devimon', '2017-05-12', 5, '1', 11);
+insert into animals 
+  (name, date_of_birth, escape_attempts, neutered, weight_kg)
+values 
+  ('Agumon', '2020-02-03', 0, '1', 10.23),
+  ('Gabumon', '2018-11-15', 2, '1', 8),
+  ('Pikachu', '2021-01-7', 1, '0', 15.04),
+  ('Devimon', '2017-05-12', 5, '1', 11);
+
+insert into animals 
+	(name, date_of_birth, escape_attempts, neutered, weight_kg)
+values 
+	('Charmander', '2020-02-08', 0, '0', 11),
+	('Plantmon', '2021-11-15', 2, '1', 5.7),
+	('Squirtle', '1993-04-02', 3, '0', 12.13),
+	('Angemon', '2005-06-12', 1, '1', 45),
+	('Boarmon', '2005-06-07', 7, '1', 20.4),
+	('Blossom', '1998-10-13', 3, '1', 17),
+	('Ditto', '2022-05-14', 4, '1', 22);
```

Comment:
- [ ] As mentioned by the previous reviewer and for the project requirements, kindly ensure that  you inserte correct values for the Animal data as some values of the `weight_kg` column are negative and other are positive.
![Screenshot from 2023-07-19 16-06-18](https://github.com/akezeth/Vet-clinic-database/assets/83042742/548c6fe6-0604-4e27-9b07-17b7ded55cde)